Not available on all arches we support and is only needed for
a few consumers.

https://bugs.gentoo.org/828482

From: Sam James <sam@gentoo.org>
Date: Wed, 8 Dec 2021 00:31:01 +0000
Subject: [PATCH] Drop greenlet dependency

--- a/setup.cfg
+++ b/setup.cfg
@@ -36,11 +36,8 @@ package_dir =
 	=lib
 install_requires = 
 	importlib-metadata;python_version<"3.8"
-	greenlet != 0.4.17;python_version>='3' and (platform_machine=='aarch64' or (platform_machine=='ppc64le' or (platform_machine=='x86_64' or (platform_machine=='amd64' or (platform_machine=='AMD64' or (platform_machine=='win32' or platform_machine=='WIN32'))))))
 
 [options.extras_require]
-asyncio = 
-	greenlet!=0.4.17;python_version>="3"
 mypy = 
 	mypy >= 0.910;python_version>="3"
 	sqlalchemy2-stubs
diff --git a/setup.cfg b/setup.cfg
index b7f69e9..91993f7 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -57,7 +57,6 @@ oracle =
 postgresql = psycopg2>=2.7
 postgresql_pg8000 = pg8000>=1.16.6
 postgresql_asyncpg = 
-	%(asyncio)s
 	asyncpg;python_version>="3"
 postgresql_psycopg2binary = psycopg2-binary
 postgresql_psycopg2cffi = psycopg2cffi
@@ -65,13 +64,10 @@ pymysql =
 	pymysql;python_version>="3"
 	pymysql<1;python_version<"3"
 aiomysql = 
-	%(asyncio)s
 	aiomysql;python_version>="3"
 asyncmy = 
-	%(asyncio)s
 	asyncmy>=0.2.3;python_version>="3"
 aiosqlite = 
-	%(asyncio)s
 	aiosqlite;python_version>="3"
 	typing_extensions!=3.10.0.1
 sqlcipher = 
